pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -335,7 +330,7 @@ func (m *manager) GetPodTopologyHints(pod *v1.Pod) map[string][]topologymanager.
 }
 
 func (m *manager) GetAllocatableCPUs() cpuset.CPUSet {
-	return m.allocatableCPUs.Clone()
+	return m.policy.GetAllocatableCPUs(m.state)
```

Comment:
don't you have to return a Copy as it was doing before?

Reply:
Thanks for reminding.